bentoml/cli/click_utils.py

```diff
@@ -188,15 +188,10 @@ def parse_bento_tag_callback(ctx, param, value):  # pylint: disable=unused-argum
 
 
 def parse_bento_tag_list_callback(ctx, param, value):  # pylint: disable=unused-argument
+    if value is None:
+        return None
     bento_tags = value.split(",")
     bento_tags = list(map(str.strip, bento_tags))
-    for bento_tag in bento_tags:
-        if not _is_valid_bento_tag(bento_tag):
```

Comment:
instead of removing this, move `parse_bento_tag_list_callback` to next to the delete command, rename it as `parse_remove_target_argument`, and change the code here to something like:

```
if _is_valid_bento_tag(remove_target) or _is_valid_bento_name(remove_target):
    return
else:
    raise click.BadParameter(...)
```